### DISCUSSION TITLE
Understanding RDKit descriptors

### DISCUSSION THREAD

**Discussion by lukasschueler** [rdkit/rdkit#7506] [FAQ]
Dear Community, 
\
\
I'm currently working on using [explainable A.I.](https://docs.abzu.ai/) in the analysis of siRNA structures and have started incorporating the descriptors provided by RDKit into my models to make clearer predictions about things like e.g. specificity, activity, or toxicity. Since the symbolic A.I. system discloses which features/descriptors it uses to create these predictions, I'm naturally confronted with interpreting the semantics behind those.
\
The article by [@greglandrum](https://github.com/greglandrum) (link below) focusing on the VSA descriptors was very helpful to better understand the dynamics of these, but I would like to broaden this question beyond the VSA descriptors. I'm aware that explaining the components of a n-dimensional chemistry space is not trivial, but **maybe this thread could be used as a place to gather experiences, understanding, or intuition by people from various fields that are using the RDKit descriptors for their research.** 

Any kind of feedback, source, or material is welcome!

I have seen this [discussion](https://github.com/rdkit/rdkit/discussions/6179), but I don't think one-liners will suffice for the topics to be illustrated. 

### Current materials

- RDKit blog: [What are the VSA-descriptors?](https://greglandrum.github.io/rdkit-blog/posts/2023-04-17-what-are-the-vsa-descriptors.html)
- [Introduction of the descriptors](https://web.archive.org/web/20190218182414/http://www.chemcomp.com/journal/vsadesc.htm) by Paul Labute
- [Gasteiger and Marsili (1980)](https://www.sciencedirect.com/science/article/abs/pii/0040402080801682) on Partial Equalization of Orbital Electronegativity
- [List of available descriptors](https://www.rdkit.org/docs/GettingStartedInPython.html#list-of-available-descriptors) with some short descriptions
- [Wildman and Crippen (1999)](https://pubs.acs.org/doi/10.1021/ci990307l) "Prediction of Physicochemical Parameters by Atomic Contributions"
- [Hall and Kier (1991)](https://onlinelibrary.wiley.com/doi/abs/10.1002/9780470125793.ch9) "The Molecular Connectivity Chi Indexes and Kappa Shape Indexes in Structure-Property Modeling"
- [Thread](https://github.com/rdkit/rdkit/discussions/6721) on _FpMorganDensity_

\
\
\
_Kindly, 
Lukas (Bioinformatics)_

**Comment by mirfanmalik** [comment]
nice work if you need something in collaboration ask me
